Source/Evolve/WarpXEvolve.cpp

```diff
@@ -120,6 +121,7 @@ WarpX::Evolve (int numsteps)
             FillBoundaryAux(guard_cells.ng_UpdateAux);
```

Comment:
Note: this call to `FillBoundaryAux` before `UpdateAuxilaryData` does not seem intuitive. Would it be safe to remove it? Does it make a difference in specific scenarios, for example, in simulations with more than one refinement level?